app/views/contacts/index.html.slim

```diff
@@ -0,0 +1,29 @@
+- content_for :heading
+  = "Address book"
+
+= render partial: 'layouts/header'
+
+.grid-row
+  .column-full.button-holder
+    = link_to 'Add new address', new_contact_path, class: 'button'
+
+table
```

Comment:
Please check if mobile works, if not, see if adding class "report" helps to table element.

Reply:
That's brilliant thanks Jav looks much better on mobile now as it collapses to a nicely formatted table.